corehq/apps/data_dictionary/views.py

```diff
@@ -45,6 +46,33 @@ def data_dictionary_json(request, domain, case_type_name=None):
     return JsonResponse({'case_types': props})
 
 
+@login_and_domain_required
+@toggles.DATA_DICTIONARY.required_decorator()
+def update_case_property(request, domain):
```

Comment:
Since this just updates the description could change the name to `update_case_property_description` (unless the plan is to use this to update other fields too)